ONSdigital/eq-author-app
# Ear 1393 change end of section to logical target

What is the context of this PR?

New Routing Rules to handle the Logical End of Section rules in v2 publisher.

How to review

* Add a question.
* Add a rule that routes to the end of current section.
* Check that it routes to the next logical section.
* Move the question into another section and check that it routes to the next logical section.
 

What to do after everything is green
    *   Bring this branch up-to-date with Origin/Master
    *   If there are a lot of commits or some are not helpful to read, squash them down
    *   Click the Merge button on this pull request
    *   Does this change mean the Capability examples questionnaire should be updated?
    *   Move the Jira ticket for this task into the next stage of the process
